dev/requirements_development.txt

```diff
@@ -7,5 +7,6 @@ pytest==6.0.2
 pytest-cov==2.10.1
 pyyaml==5.3.1
 pip-check==2.6
+autoflake==1.4
```

Comment:
What does this tool do and where is it used?

Reply:
It removes unneeded import which are reported by flake. It was really annoying to remove them all by hand... It's used in the Makefile